src/test/java/com/meltmedia/jgroups/aws/WithEc2.java

```diff
@@ -0,0 +1,26 @@
+package com.meltmedia.jgroups.aws;
+
+import com.amazonaws.services.ec2.AmazonEC2;
+import com.amazonaws.services.ec2.model.*;
+
+import java.util.Arrays;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public interface WithEc2 {
+  default AmazonEC2 ec2Mock(Tag... tags) {
```

Comment:
Yes ;-) This is a habit (good or bad) that I have adopted for tests lately... guess it's my dream of mixins/traits that leaks. It becomes quite handy in larger projects, where you can have a lot of cross cutting test concerns... But If you think it's to wacky, I can always change :-)